docs/conf.py

```diff
@@ -204,6 +204,7 @@
     "sunkit_image": ("https://docs.sunpy.org/projects/sunkit-image/en/stable/", None),
     "sunkit_instruments": ("https://docs.sunpy.org/projects/sunkit-instruments/en/stable/", None),
     "zeep": ("https://docs.python-zeep.org/en/stable/", None),
+    "contourpy": ("https://contourpy.readthedocs.io/en/stable/", None),
```

Comment:
Is this correct? the ref is still not working?

Reply:
What is not working?